### ISSUE TITLE
No indication of where tooltip is being used when using a screen reader

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#1878]
This ticket also came from a conversation had with Bill Tipton. He told us that anywhere there is a tooltip, it should be expected that the screen reader describes that. We came across this problem when we had Bill tab over to our switch to light/dark mode button on the design system home page. We should research on how we should enhance that in the back end.
